app/assets/stylesheets/navbar.css

```diff
@@ -21,13 +25,13 @@
         display: none;
     }
 
-    .navbar.navbar-fixed-top {
+    .navbar {
+        height: 64px !important;
         background-color: #0f3157 !important;
```

Comment:
Can you move this out of the `media` selector because it's a duplicate to line 11?